### DISCUSSION TITLE
Support for <Profiler> component

### DISCUSSION THREAD

**Discussion by akaltar** [preactjs/preact#3991] [Ideas]
Are there any plans for supporting the [Profiler](https://react.dev/reference/react/Profiler) Component in `preact/compat`?

Since Preact is very performance focused, it would make it easier to evaluate how much of a performance benefit does switching to Preact from React have.

Or are there any workarounds that I might be unaware of?

Thanks

**Comment by marvinhagemeister** [comment]
There are no plans to add something like React's `Profiler` component. Reason is that interest is very low and that it's not a reliable way to compare frameworks. Just the mere fact of injecting additional measuring logic inside the application can skew the results. The best way to compare frameworks is to use the "Performance" tab in Chrome DevTools. It's a sampling profiler that attaches to the engine, rather then modifying user code. That's much more accurate to compare frameworks or any JavaScript code for that matter.

**Comment by akaltar** [comment]
Thank you for the super quick response
